sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 # Release History
 ## 5.1.0-beta.1 (Unreleased)
-
-## 1.1.0-beta.1 (Unreleased)
 **New features**
 - Text analytics SDK update the service to version `v3.1-preview.1` from `v3.0`.
-- Added Opinion Mining feature.
+- Added support for the Opinion Mining feature.
+
+## 5.0.0 (2020-07-27)
```

Comment:
Why is this in the changeset ?
It should have been a part of the increment PR?

Reply:
it is because it is not against the latest of master. It should be resolved.